src/script/integration/IntegrationRepository.ts

```diff
@@ -210,12 +236,12 @@ export class IntegrationRepository {
   /**
    * Remove service from conversation.
    *
-   * @param {Conversation} conversationEntity - Conversation to remove service from
-   * @param {User} userEntity - Service user to be removed from the conversation
-   * @returns {Promise} Resolves when service was removed from the conversation
+   * @param conversationEntity Conversation to remove service from
+   * @param userEntity Service user to be removed from the conversation
+   * @returns Resolves when service was removed from the conversation
    */
-  removeService(conversationEntity, userEntity) {
-    const {id: userId, serviceId} = userEntity;
+  removeService(conversationEntity: Conversation, userEntity: User): Promise<any> {
+    const {id: userId, serviceId} = userEntity as any;
```

Comment:
Let's connect our brains to find a solution on how to dissolve `any`.

Reply:
`User` is not typed yet so it doesn't make sense to create an interface here.